source/conf.py

```diff
@@ -201,15 +201,10 @@
 
 # -- Options for intersphinx extension ---------------------------------------
 
-# Example configuration for intersphinx: refer to the Python standard library.
-intersphinx_mapping = {'https://docs.python.org/': None}
+# Intersphinx stub configuration
+intersphinx_mapping = {}
```

Comment:
Just some feedback: I think intersphinx was once relevant because we planned to host separate docs repositories for different projects.